members_and_sponsors.md

```diff
@@ -33,3 +33,6 @@ Members with affiliations:
 - Athan Reines - Quansight
 - Saul Shanabrook - Quansight
 - Keith Kraus - cuDF
+- Hyukjin Kwon - PySpark, Koalas
```

Comment:
Looks it's written as "Apache Arrow" instead of "pyarrow". Maybe we should keep it consistent with "Apache Spark" or switching "Apache Arrow" to "pyarrow"?
WDYT @gatorsmile and @ueshin?

Reply:
Oh yes, good point @HyukjinKwon. PySpark isn't really a separate project right? Sticking with project names only (so Apache Spark) sounds good to me.

Reply:
Yes, Apache Spark is more accurate. :-)